provision/rosa-cross-dc/Taskfile.yaml

```diff
@@ -420,6 +434,12 @@ tasks:
           ROSA_CLUSTER_NAME: "{{.ROSA_CLUSTER_NAME_2}}"
           KC_HOSTNAME_OVERRIDE: "{{.KC_CLIENT_URL}}"
           KC_HEALTH_HOSTNAME: "{{.KC_HEALTH_URL_CLUSTER_2}}"
+      - task: create-env-configmap
```

Comment:
Is this environment config map per cluster or per namespace in the ROSA cluster ? Would it uniquely identify a particular deployment on the cluster ?

Reply:
or is our assumption to have only one deployment per cluster during these benchmark tests ?

Reply:
the env config map is created on the cluster per namespace. So it is unique for each *-keycloak namespace, and it is possible to run the benchmark tests on different namespaces and we will have the generated JSON with corresponding data.

Reply:
perfect, thanks for letting me know.